common/changes/@uifabric/charting/PrograssBarChart_2018-07-24-09-13.json

```diff
@@ -2,7 +2,7 @@
   "changes": [
     {
       "packageName": "@uifabric/charting",
-      "comment": "Adding Stacked Bar chart",
+      "comment": "Change import path for package",
       "type": "patch"
```

Comment:
When you add code, you are making a "minor" release, not a patch.

A patch is something that you can go back a version and nothing will break contract wise.

A minor release is when a feature is added, such that going back a version would break a contract. Adding a component, adding a method, adding an parameter... All would break going back 1 version.